docs/metadata/3.9.0.jsx

```diff
@@ -126,7 +126,7 @@ const entries = [
   {
     id: "vertx-web-api-service",
     name: "Web API Service",
-    description: "Directly connect your OpenAPI 3 web routers to event-bus services.",
+    description: "Helps you handling HTTP Requests using the Vert.x Event Bus.",
```

Comment:
The grammar is incorrect. It should be `Helps you handle HTTP requests using the Vert.x Event Bus.`

Reply:
```suggestion
    description: "Helps you handle HTTP requests using the Vert.x event bus.",
```